internal/cmd/testdata/lint/allgroup/prototool.yaml

```diff
@@ -1,3 +1,14 @@
 protoc_include_wkt: true
 lint:
-  group: all
```

Comment:
Ah shoot, this is why I had it be a soft failure if `lint.group` is specified - this test tests if all linters are turned on. I'm not sure what to do here - we could have all these options be an error if not in devel mode? But then you have to propagate development mode all the way down, which I don't like...I don't know ugh.

I'm moderately confused as to how tests are passing if you remove this.

Reply:
I fixed the test by adding all of the missing linters under `include_ids`. This way, we mimic the `all` setting because it's essentially: `Default + (All - Default) == All`